src/status_im/contexts/centralized_metrics/core.cljs

```diff
@@ -0,0 +1,88 @@
+(ns status-im.contexts.centralized-metrics.core
+  (:require
+    [legacy.status-im.utils.build :as build]
+    [native-module.core :as native-module]
+    [re-frame.interceptor :as interceptor]
+    [react-native.platform :as platform]
+    [taoensso.timbre :as log]
+    [utils.re-frame :as rf]))
+
```

Comment:
It would be good to have a bit more information/documentation on how the metrics are supposed to work instead of only the code. For example, how the developer can test the providers.

Reply:
should i add that in status-go?

Reply:
In status-go would be the best 👍🏼